docs/how_to/point_manipulator.md

```diff
@@ -8,28 +8,29 @@ Precisely positioning a point in 3D is a useful way to annotate specific positio
 
 ## Instructions
 
-1. Open the [points_manipulator.py](https://napari-threedee.github.io/generated/gallery/plugin/points_manipulator_plugin/) example.
+1. To run this example, first download the script from the [Examples Gallery page](https://napari-threedee.github.io/generated/gallery/plugin/points_manipulator_plugin/); the link is at the bottom. Ensure the location is somewhere memorable. Or, if you've cloned this repository, the script can be found at `napari-threedee/docs/examples/plugin/points_manipulator_plugin.py`. 
 
+2. Ensure you have the appropriate virtual environment active, change directories to the script location, and then you can execute the script using:
+	```bash
+	python points_manipulator_plugin.py
+	``` 
 
-    ![example opened](https://user-images.githubusercontent.com/1120672/207381513-8fd3f7e9-157f-4ef9-99c1-551bd329842b.png)
-   The example opens in 3D rendering mode. Click and drag to rotate the view.
-   
-2. Open the point manipulator plugin from the Plugins menu: Plugins -> napari-threedee -> point manipulator. Click the "activate" to start the plugin.
+3. The viewer will open in 3D rendering mode with a Points layer and 3 points and the widget already visible. The widget can be otherwise accessed from the Plugins menu: **Plugins -> napari-threedee -> point manipulator**. Click the "activate" to start the plugin.
 
 	![plugin opened](https://user-images.githubusercontent.com/1120672/207382282-dad2bd6f-68cf-47d6-89ed-be326d320f93.png)
 
-4. Set the Points later to "Selection" mode by clicking the "Selection" button in the layer controls
+4. Set the Points later to "Selection" mode by clicking the "Selection" button in the layer controls.
```

Comment:
From the text it's ambiguous whether the "Selection" button refers to something in the napari-threedee widget, or in napari itself. I think we need to make it clearer that we need to enter the napari point selection mode, which is only accessible if you have a points layer selected in the layerlist, and is enabled from one of the top row buttons in the layer controls in the top left corner of napari. The fact this button is just an icon, and doesn't have any words until you hover over it makes it important to explain clearly.

Reply:
I believe we have a screenshot with the button highlighted in the docs. Do you think that is sufficient @GenevieveBuckley 

![Screenshot 2024-06-07 at 18 28 29](https://github.com/napari-threedee/napari-threedee/assets/1120672/ea4091ab-06cd-4133-be47-07fd19584922)

Also minor typo fix:

```suggestion
4. Set the Points layer to "Selection" mode by clicking the "Selection" button in the layer controls.
```

Reply:
There is only a Points layer here, so it will be selected. I think with the visual it's fine?
I think we can assume people have some understanding of the napari GUI if they have gotten to the point of using this quite advanced plugin.